docs/platforms/android/session-replay/index.mdx

```diff
@@ -71,6 +71,16 @@ While you're testing, we recommend that you set `sessionSampleRate` to `1.0`. Th
 
 Once testing is complete, **we recommend lowering this value in production**. We still recommend keeping `onErrorSampleRate` set to `1.0`.
 
+## User Session
+
+A user session starts when the Sentry SDK is initialized or when the application enters the foreground. The session will capture screen transitions, navigations, touches and other events until the application is sent to the background. If the application is brought back to the foreground within 30 seconds (default), the same `replay_id` will be used and the session will continue. Otherwise, the session will be terminated according to the rules for [Release Health](/platforms/android/configuration/releases/#sessions). You can adjust the `sessionTrackingIntervalMillis` option to extend or shorten the duration of a single replay, depending on your needs.
```

Comment:
"Otherwise, the session will be terminated according to the rules for [Release Health](/platforms/android/configuration/releases/#sessions)" I don't see anything in the rules for RH link that isn't already mentioned in this paragraph, can we move this? or mention the rules for RH directly